.circleci/config.yml

```diff
@@ -625,12 +625,7 @@ workflows:
     - integration:
         name: test/e2e
         requires:
-          - minikube_v1_13_0
-          - minikube_v1_14_0
-          - minikube_v1_15_0
-          - minikube_v1_16_0
-          - minikube_v1_17_0
-          - minikube_v1_18_2
+          - images
```

Comment:
I don't mind it. My only concern was that we're using shared resources (I think). Meaning that the more parallel test we try to push, the higher the chances for flakiness are. I am OK with this, let's see how this will play and we can move it if we decide later.

Reply:
Yes, let's observe how it will behave. Minikube jobs are flaky itself and often we don't see e2e results